mappings/net/minecraft/block/RedstoneWireBlock.mapping

```diff
@@ -7,8 +7,15 @@ CLASS net/minecraft/class_2457 net/minecraft/block/RedstoneWireBlock
 	FIELD field_11439 WIRE_CONNECTION_WEST Lnet/minecraft/class_2754;
 	FIELD field_11440 WIRE_CONNECTION_NORTH Lnet/minecraft/class_2754;
 	FIELD field_24413 DOT_SHAPE Lnet/minecraft/class_265;
+	FIELD field_24414 LINE_SHAPE Ljava/util/Map;
+	FIELD field_24415 LINE_SIDE_SHAPE Ljava/util/Map;
+	FIELD field_24416 outlineShapes Ljava/util/Map;
+	FIELD field_24466 WIRE_COLOR [Lnet/minecraft/class_1160;
 	FIELD field_24733 dotState Lnet/minecraft/class_2680;
-	METHOD method_10477 getRenderConnectionType (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Lnet/minecraft/class_2773;
+	METHOD method_10477 getRenderConnectionSideType (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_2350;)Lnet/minecraft/class_2773;
```

Comment:
What does this method do? Inserting a "side" in the name doesn't make it more clear. Also afaik now mojang doesn't do client specific render states any more but compute it on server and send it to client, so I doubt the "render" part of the name as well.